db/README.md

```diff
@@ -10,6 +10,8 @@ Once data is processed and stored in the Cloud SQL database, corresponding views
 
 You can use the scripts in this directory (`db`, `run-proxy`) to connect and interact with the Cloud SQL instance locally.
 
+`db` depends on `mysql` v8.
```

Comment:
Other items to be updated (in another PR):
* Update or delete --sql option: `db -s 'SHOW DATABASES'` does not work.
* Software to installed locally: brew mysql@8 etc, mysqldump, mysqlshow